serverless/src/forwarder.ts

```diff
@@ -208,7 +209,19 @@ export async function getExistingLambdaLogGroupsOnStack(cloudWatchLogs: CloudWat
 
 export async function shouldSubscribeLogGroup(cloudWatchLogs: CloudWatchLogs, logGroupName: string) {
   const subscriptionFilters = await describeSubscriptionFilters(cloudWatchLogs, logGroupName);
-  return subscriptionFilters.length === 0;
+  let foundDatadogSubscriptionFilter = false;
+  const numberOfActiveSubscriptionFilters = subscriptionFilters.length;
+  if (numberOfActiveSubscriptionFilters === MAX_ALLOWABLE_LOG_GROUP_SUBSCRIPTIONS) {
```

Comment:
What if the number of subscription filters is greater than the number of maximum allowable log group subscriptions? Could happen in the future if AWS changes the maximum on their side.

Reply:
Hmmm yeah that could happen if AWS updates and if Datadog does not update the macro, good catch, thanks